mobile/mobile_test.go

```diff
@@ -103,11 +105,11 @@ func (tm *testMessenger) Notify(event *Event) {
 }
 
 func TestNewWallet(t *testing.T) {
-	var err error
-	testVars.recovery, err = NewWallet(12)
+	wallet, err := wallet.WalletFromWordCount(12)
```

Comment:
This is just testing the wallet... whereas we want to test the mobile API here, right?

Reply:
it's more about bootstrapping the recovery passphrase for the next tests

Reply:
Unfortunately, mobile needs to provide an additional `NewWallet` method that can be exposed to the native SDKs, which is why this existed here. If it lived in the wallet package, we'd have to run `gobind` over it as well. 

ref -> https://github.com/textileio/photos/blob/master/ios/Pods/TextileCore/Mobile.framework/Versions/A/Headers/Mobile.objc.h#L288

and 

https://github.com/textileio/photos/blob/master/ios/Pods/TextileCore/Mobile.framework/Versions/A/Headers/Mobile.objc.h#L293

This will break the framework as is.

Reply:
Okay, makes sense. Will reinstate.